dts/bindings/spi/nxp,imx-flexspi.yaml

```diff
@@ -13,3 +13,50 @@ properties:
 
     interrupts:
       required: true
+
+    ahb-base:
+      type: int
+      required: true
+      description: |
+        AHB address for read/write access to serial flash memory.
```

Comment:
How is this address different from the MMIO region that the flash controller specifies?

Reply:
The MMIO region specified by the flash controller contains the controller registers, this is the flash itself memory mapped into the AHB address space.